.env.template

```diff
@@ -4,4 +4,5 @@ GOVUK_NOTIFY_API_KEY=
 DOMAIN=http://localhost:3000
 GOVUK_APP_DOMAIN=http://localhost:3000
 GOVUK_WEBSITE_ROOT=http://localhost:3000
-GIAS_API_ROOT=https://ea-edubase-api-prod.azurewebsites.net
\ No newline at end of file
+GIAS_API_ROOT=https://ea-edubase-api-prod.azurewebsites.net
+REGISTER_AND_PARTNER_URL=http://localhost:3001
```

Comment:
pls tell your editor to start ending files with new lines (or, ending every line with an End Of Line character) lol

If you install editorconfig it does it for you: https://github.com/DFE-Digital/ecf-engage-and-learn/blob/develop/.editorconfig